package.json

```diff
@@ -26,11 +26,13 @@
     "@oclif/config": "^1",
     "@oclif/plugin-help": "^2",
     "axios": "^0.19.0",
+    "codecov": "^3.5.0",
```

Comment:
This lib should be DEV dependency.

Reply:
So should I move it the devDependencies?

Reply:
yep